org-code-javabuilder/lib/build.gradle

```diff
@@ -25,6 +25,8 @@ dependencies {
     implementation group: 'com.amazonaws', name: 'aws-java-sdk-apigatewaymanagementapi', version: '1.11.978'
     // https://mvnrepository.com/artifact/com.amazonaws/aws-java-sdk-sqs
     implementation group: 'com.amazonaws', name: 'aws-java-sdk-sqs', version: '1.11.978'
+    // https://mvnrepository.com/artifact/org.json/json
+    implementation group: 'com.googlecode.json-simple', name: 'json-simple', version: '1.1.1'
```

Comment:
This doesn't look like the package matches the comment associated with it. Did you intend to use json-simple or org.json?

Reply:
I meant to use json-simple, I'll replace that comment

Reply:
Follow-up question: I also added a different JSON library on concurrent change. I was planning to swap over to com.googlecode.json-simple, but when I looked up the package, it looked somewhat unmaintained. i.e. the last version was from 2012 and it's hosted in google's code archive. It looks like you might have tried out org.json at first. Was there anything in particular that made you swap?

Reply:
I think I just found more examples for json-simple, I can switch over to org.json. Can you link the change where you added org.json so I can see how it works?

Reply:
There's a couple unrelated changes here, but ClientMessage.java is where I use org.json: https://github.com/code-dot-org/javabuilder/pull/32/commits/5521818e6846fb7e7a5ecb23d16ef176feae4d96#diff-bf69cb62b03f4ac01a380376939981a2d0db735a5b0a9c42d390fef9380cd017
^ This particular commit moves from using org.json to json-simple